client/components/header-component.js

```diff
@@ -2,22 +2,44 @@ import AvatarDropdown from "./avatar-dropdown-component.js";
 import { fetchUserById } from "../src/data-access/api-calls/calls.js";
 
 const HeaderComponent = {
-  components: { AvatarDropdown },
-  template: `<div class="header-container">
-    <div class="brand">
-        <a href="/pages/homepage/homepage.html" class="logo">
-        <img src="/assets/logo_pc_full.png" alt="logo" width="90" height="70"/>
-        </a>
-        </div>
-        <div class="nav-bar">
-            <a class="nav-link" href="/pages/projects/projects.html">projects</a>
-            <a class="nav-link" href="/pages/members/members.html">members</a>
-            <a class="nav-link" href="/pages/about-us/about-us.html">about us</a>
-            <a class="nav-link" href="/pages/contact-us/contact-us.html">contact us</a>
-        </div>
+  components: {    AvatarDropdown,  },
+  template: `
+    <div class="header-container">
+      <div class="brand">
+        <a href="/pages/homepage/homepage.html" class="logo"><img src="/assets/hyh-connect-logo-header.png" alt="logo" height="80"/></a>
+      </div>
+      <div class="nav-bar header-menu-horizontal">
+          <a class="nav-link" href="/pages/projects/projects.html">projects</a>
+          <a class="nav-link" href="/pages/members/members.html">members</a>
+          <a class="nav-link" href="/pages/about-us/about-us.html">about us</a>
+          <a class="nav-link" href="/pages/contact-us/contact-us.html">contact us</a>
+      </div>
+      <div class="header-menu-avatar">
         <avatar-dropdown :avatar="avatar" v-if="isLoggedIn"></avatar-dropdown>
         <a class="sign-in-btn" href="/pages/login/login.html" v-else>sign in</a>
-        </div>`,
+      </div>
+      <div class="header-menu-burger">
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" 
+              data-bs-target="#header-menu-burger">
+          <span class="header-burger_line header-burger_line_first"></span>
+          <span class="header-burger_line header-burger_line_second"></span>
+          <span class="header-burger_line header-burger_line_third"></span>
+        </button>
+        <div id="header-menu-burger" class="collapse navbar-collapse header-menu-vertical">
+          <button class="navbar-toggler header-burger-close" type="button" data-bs-toggle="collapse" 
+              data-bs-target="#header-menu-burger">
+            <span class="header-menu-close-line"></span>
+            <span class="header-menu-close-line"></span>
+          </button>
+          <a class="burger-link" href="/pages/projects/projects.html">projects</a>
+          <a class="burger-link" href="/pages/members/members.html">members</a>
+          <a class="burger-link" href="/pages/about-us/about-us.html">about us</a>
+          <a class="burger-link" href="/pages/contact-us/contact-us.html">contact us</a>
+          <a class="burger-link" href="/pages/login/login.html">sign in</a>
+          <a class="burger-link" href="/pages/user-profile/user-profile.html">sign out</a>
```

Comment:
Sign out should not be visible if not signed in.
There should also a method be attached to clear the token